management/sql/entities.go

```diff
@@ -0,0 +1,78 @@
+package sql
+
+import (
+	"encoding/xml"
+)
+
+// DatabaseServerCreateParams represents the set of possible parameters
+// when issuing a database server creation request to Azure.
+//
+// https://msdn.microsoft.com/en-us/library/azure/dn505699.aspx
+type DatabaseServerCreateParams struct {
+	XMLName                    xml.Name `xml:"http://schemas.microsoft.com/sqlazure/2010/12/ Server"`
+	AdministratorLogin         string
+	AdministratorLoginPassword string
+	Location                   string
+	Version                    string
+}
+
+// DatabaseServerCreateResponse represents the response following the creation of
+// a database server on Azure.
+type DatabaseServerCreateResponse struct {
+	ServerName string
+}
+
+const (
+	DatabaseServerVersion11 = "2.0"
+	DatabaseServerVersion12 = "12.0"
+)
+
+// DatabaseServer represents the set of data recieved from
+// a database server list operation.
+//
+// https://msdn.microsoft.com/en-us/library/azure/dn505702.aspx
+type DatabaseServer struct {
+	Name                     string
+	AdministratorLogin       string
+	Location                 string
+	FullyQualifiedDomainName string
+	Version                  string
+	State                    string
+}
+
+type ListDatabaseServersResponse struct {
+	DatabaseServers []DatabaseServer `xml:"Server"`
```

Comment:
"Server" should be "ServiceResource" otherwise it will not work:

http://play.golang.org/p/4_tpfVF6dK it doesn't match the response.